rollup.config.js

```diff
@@ -1,69 +1,46 @@
-import resolve from '@rollup/plugin-node-resolve';
-import commonjs from '@rollup/plugin-commonjs';
-import typescript from '@rollup/plugin-typescript';
-import dts from 'rollup-plugin-dts';
-import postcss from 'rollup-plugin-postcss';
-import image from 'rollup-plugin-image';
-import copy from 'rollup-plugin-copy-assets';
-import svg from 'rollup-plugin-svg';
+import resolve from "@rollup/plugin-node-resolve";
+import commonjs from "@rollup/plugin-commonjs";
+import typescript from "@rollup/plugin-typescript";
+import dts from "rollup-plugin-dts";
+import postcss from "rollup-plugin-postcss";
+import image from "@rollup/plugin-image";
+import copy from "rollup-plugin-copy-assets";
+import url from "@rollup/plugin-url";
+import peerDepsExternal from "rollup-plugin-peer-deps-external";
 
-import peerDepsExternal from 'rollup-plugin-peer-deps-external';
-
-const packageJson = require('./package.json');
+const packageJson = require("./package.json");
 
 export default [
   {
-    input: 'src/index.ts',
+    input: "src/index.ts",
     output: [
       {
         file: packageJson.main,
-        format: 'cjs',
+        format: "cjs",
         sourcemap: true,
       },
       {
         file: packageJson.module,
-        format: 'esm',
+        format: "esm",
         sourcemap: true,
       },
     ],
-    externals: {
-      react: {
-        root: 'React',
-        commonjs2: 'react',
-        commonjs: 'react',
-        amd: 'react',
-        umd: 'react',
-      },
-      'react-dom': {
-        root: 'ReactDOM',
-        commonjs2: 'react-dom',
-        commonjs: 'react-dom',
-        amd: 'react-dom',
-        umd: 'react-dom',
-      },
-    },
-    module: {
-      rules: [
-        // all files with a `.ts` or `.tsx` extension will be handled by `ts-loader`
-        { test: /\.tsx?$/, loader: 'ts-loader' },
-      ],
-    },
     plugins: [
       resolve(),
       commonjs(),
-      typescript({ tsconfig: './tsconfig.json' }),
+      typescript({ tsconfig: "./tsconfig.json" }),
       postcss(),
       image(),
-      svg(),
+      url(),
       peerDepsExternal(),
       copy({
-        assets: ['src/assets'],
+        assets: ["src/assets"],
       }),
     ],
   },
   {
-    input: 'dist/esm/types/index.d.ts',
-    output: [{ file: 'dist/index.d.ts', format: 'esm' }],
+    input: "dist/esm/types/index.d.ts",
```

Comment:
lot of changes due to " and ',
I think we can setup a pettier or .editorconfig to prevent the issue